tools/RunVersionController.ps1

```diff
@@ -0,0 +1,13 @@
+if (!(Test-Path "C:/Program Files/PowerShell/Modules/PowerShellGet"))
```

Comment:
@maddieclayton to avoid having a separate script to run the version controller, I think it would be better to do this check for `PowerShellGet` at the beginning of the `Main` method in the version controller, and fail if the module cannot be found with an error message that tells the user to run the below command in admin mode.